### ISSUE TITLE
Allow configuration of event reminder and/or alarms

### ISSUE THREAD

**Issue by tamtashi** [calcom/cal.com#12247]
### Is your proposal related to a problem?

Unfortunately, the events in the calendar do not have reminder/alarms set. As a result, I miss some of these meetings because I do not manually edit each one. Please add the ability to add one or more reminder/alarms to each event.

### Describe the solution you'd like

For each event I want to configure one or many reminder/alarms that remind me of a scheduled meeting on my phone, in Thunderbird or any other calendar client I use.

### Describe alternatives you've considered

There is now alternative a a successful configured reminder/alarm ;)

### Additional context

There is full support in ICS for reminder and alarms, should be easy to add.

### Requirement/Document

---
##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

**Comment by PeerRich** [comment]
you can use workflows for reminders

**Comment by riordan** [comment]
Hi @PeerRich and team,

I'm hoping you're open to reconsidering the feature request by @tamtashi regarding event reminders/alarms in the calendar (as discussed here and in #1540). Instead of creating a new issue, I believe it's more efficient to reconsider this existing one.

I understand the suggestion to use Cal.com Workflows for reminders. However, I haven't found a way for these workflows to modify the actual calendar invite (for the organizer, scheduler, and guests) to include an ICS reminder. Currently, I manually add in-event reminders through my calendar client for each new event. This process is cumbersome and results in sending update emails to attendees, with the hope that their calendars recognize these added reminders.

In-event reminder notifications are crucial for me and many users, serving as the primary prompt for transitioning between meetings/events. Email reminders often get overlooked, especially minutes before an event starts. The absence of built-in reminders in the initial event invitation has led to missed meetings for both myself and guests, as we expect these notifications to be part of the event invite.

Integrating this feature as a default property of the event type, rather than as a workflow, seems more appropriate. It would act as a universal setting for all attendees, enhancing the event's functionality without replacing existing workflow features. Workflows are invaluable for broader communication and context, but the simplicity and directness of in-event reminders have their unique importance.

**Comment by iandoe** [comment]
Hey @PeerRich,

I'm willing to tackle this, i have had a thorough look at the codebase and got a good grasp of what needs to be done, the ICS library supports alarms (reminders).

I love cal.com but i have missed two events because of this already, the reminders are super useful to me.

Would you reconsider and approve work on this ? I'd hate to submit a PR and have it rejected off-hand.

Thanks.
